Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM python:3.11
+
+ENV PATH=$PATH:/root/.local/bin
+RUN pip install pipx && pipx install poetry && poetry config virtualenvs.create false
+
+WORKDIR /root/project
+
+# Copy only dependency-related files for caching
+COPY poetry.lock pyproject.toml /root/project/
+RUN poetry install --no-root
+
+# Copy actual source code
+COPY README.md .
+COPY torch_ecg_volta torch_ecg_volta
```

Comment:
Not a fan of `COPY` I would rather mount the volume in the docker compose or with the `-v` command when deploying the image.